### DISCUSSION TITLE
Can I use none-ls and disable conform.nvim and nvim-lint safely without breaking any functionality of LazyVim?

### DISCUSSION THREAD

**Discussion by Nitestack** [LazyVim/LazyVim#2001] [Q&A]
Can I **use** none-ls (**alone**) and **disable** *conform.nvim* and *nvim-lint* safely **without breaking** any functionality of *LazyVim*?
Looking at the config, there is much custom code from *LazyVim* provided for both *conform.nvim* and *nvim-lint*.
If I **disable** *conform.nvim* and *nvim-lint*, does something break in functionality of *LazyVim*?

Sorry in advance if this question is stupid.

**Comment by cerebrl** [comment]
@folke does a pretty good job encapsulating each plugin's configuration, and not coupling plugins to other plugins. Plus, a lot of work has gone into the switch from `null-ls` to `none-ls` then to `conform.nvim` (there are some discussions about it here), which likely helped find instances of coupling. 

I'd say your safe to disable `conform.nvim` and `nvim-lint` without anything terrible to happen in your LazyVim installation. Being that it's [so easy to disable a plugin](https://www.lazyvim.org/configuration/plugins#-disabling-plugins), [and enable `none-ls`](https://www.lazyvim.org/extras/lsp/none-ls), I'd say go for it. If you do lose some needed functionality, you can always reverse it without much effort.

**Reply by Nitestack** [reply]
Thanks for the answer!
